changelog/fragments/empty-patch-requests.yaml

```diff
@@ -0,0 +1,4 @@
+entries:
+  - description: >
+      Filtering out empty patch requests generated from the 3-way merge to avoid unnecessary load on kube apiserver.
```

Comment:
```suggestion
      For helm-based operators, empty patch requests generated from the 3-way merge are filtered to avoid unnecessary requests to the kube apiserver.
```

Reply:
@estroz - updated as suggested.  Thanks for the suggestion.  It makes a lot of sense to call out helm-based operators.